js/controller.js

```diff
@@ -1788,6 +1785,24 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       }
     },
 
+    /**
+     * The function handles a click on multiAudio icon
+     */
+    toggleMultiAudioScreen: function() {
```

Comment:
Would it be possible to add this behavior to the `toggleScreen` instead of having a separate function for multi-audio? If doing this doesn't cause issues then I think it would be better to keep this logic generic, that we we don't have two different code paths to maintain.

Reply:
Hello @jpfranco !
Thanks for the comment!
Yes, it is possible, but behavoir of this functions is different, so in this case we will have something like this:
```
    toggleScreen: function(screen) {
      this.isNewVrVideo = false;
      if (screen === CONSTANTS.SCREEN.MULTI_AUDIO_SCREEN) {
        this.toggleMultiAudioScreen();
      } else {
        if (this.state.screenToShow === screen) {
          this.closeScreen();
        }
        else {
          if (this.state.playerState === CONSTANTS.STATE.PLAYING) {
            this.pausedCallback = function() {
              this.state.pluginsElement.addClass('oo-overlay-blur');
              this.state.screenToShow = screen;
              this.renderSkin();
            }.bind(this);
            this.mb.publish(OO.EVENTS.PAUSE);
          }
          else {
            this.state.screenToShow = screen;
            this.state.pluginsElement.addClass('oo-overlay-blur');
            this.renderSkin();
          }
        }
      }
    }
```
It is not beautiful, is not it?

Reply:
@KseniiaSokolovaEpam what I had in mind was to refactor the `toggleScreen` method in order to include the use case for multi audio, rather than call a separate `toggleMultiAudioScreen` function from within `toggleScreen`.

This is not that important though, so I'm ok with this as long as the second reviewer agrees.